tests/publiccloud/sles4sap.pm

```diff
@@ -15,7 +15,17 @@ use Mojo::Base 'publiccloud::basetest';
 use testapi;
 use Mojo::File 'path';
 use Mojo::JSON;
+use version_utils 'is_sle';
 
+# Global variables
+our $crm_mon_cmd = 'crm_mon -R -r -n -N -1';
```

Comment:
I don't like this definition here as it's not sles4sap-specific, and as we already have it in `lib/hacluster`. If the command is only used internally in `lib/sles4sap`, perhaps changing `our` for `my` would be better, but IMO it would be better to remove the definition here and call directly `assert_script_run $hacluster::crm_mon_cmd`.

Reply:
As I quickly discussed with you this doesn't work... I don't know why exactly (maybe the declaration inside of the `hacluster library is not good?) but I would like to avoid loading the library just for one function, as it causes some issues with the cleanup stuff. I can workaround these I thing, but I really would like to avoid too complicated code for such a thing. This test is already "too" complicated I would say (see also all the TF code!).

WDYT?

Reply:
Agree. Let's leave as is and then work on a solution to refactor common code here and in `lib/hacluster`.

Reply:
I used `my` instead of `our` as you first suggested, as it seems to be enough here.

Reply:
> I used `my` instead of `our` as you first suggested, as it seems to be enough here.

Yes, scope should be local.